### PR TITLE
Louisiana Onboarding

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR adds the schemas for Louisiana so we can onboard them.

## Changes
- Adds a CSV schema
- Adds an HL7 schema

## Checklist
- [ ] Tested locally?
- [ ] Added new dependencies?
- [ ] Updated the docs?
- [ ] Added a test?
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### CODE REVIEW

**Review by jimduff-usds — APPROVED** [review]

File: prime-router/metadata/schemas/LA/la-covid-19-csv.schema [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    csvFields:  [ { name: Patient State } ]
+
+  - name: patient_zip_code
+    csvFields: [ { name: Patient Zipcode } ]
+
+  - name: patient_phone_number
+    csvFields: [ { name: Patient Phone } ]
+
+  - name: ssn
+    type: TEXT
```
[the remark this answers is not in the document]
Comment (MauriceReeves-usds) [reply prime-router/metadata/schemas/LA/la-covid-19-csv.schema:38]: Yes

File: prime-router/metadata/schemas/LA/la-covid-19-csv.schema [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+  - name: test_result_units
+    csvFields: [{ name: ResultUnits }]
+
+  - name: reference_range
+    csvFields: [{ name: Reference Range }]
+
+  - name: phyname
+    mapper: 'concat(ordering_provider_last_name, ordering_provider_first_name)'
+    csvFields: [{ name: Phyname }]
```
[the remark this answers is not in the document]
Comment (MauriceReeves-usds) [reply prime-router/metadata/schemas/LA/la-covid-19-csv.schema:74]: Yes, I can

File: prime-router/metadata/schemas/LA/la-covid-19-csv.schema [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  - name: patient_gender
+    csvFields: [{ name: Sex }]
+
+  - name: patient_ethnicity
+    csvFields: [{ name: Ethnicity }]
+
+  - name: specimen_collection_date_time
+    csvFields: [{ name: DateColl, format: "YYYYMMDD" }]
+
+  - name: test_result_report_date
```
[the remark this answers is not in the document]
Comment (MauriceReeves-usds) [reply prime-router/metadata/schemas/LA/la-covid-19-csv.schema:54]: Yes, I think that will do.

**Comment by MauriceReeves-usds** [comment]
A file was sent to LA. Waiting for feedback from them now.

**Pull request merged**